docs/advanced_topics/documents/index.md

```diff
@@ -1,9 +1,9 @@
-Documents
-=========
-
+# Documents
 
+```{eval-rst}
 .. toctree::
     :maxdepth: 2
 
     custom_document_model
     title_generation_on_upload
+```
```

Comment:
This can probably use the myST block syntaxe instead of eval-rst

Here is an example from another page. https://raw.githubusercontent.com/wagtail/wagtail/main/docs/getting_started/index.md

```{toctree}
---
maxdepth: 1
---
tutorial
demo_site
integrating_into_django
the_zen_of_wagtail
```